FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentGcArmyExpedition.cs

```diff
@@ -0,0 +1,31 @@
+﻿using FFXIVClientStructs.FFXIV.Client.System.String;
+using FFXIVClientStructs.FFXIV.Component.GUI;
+
+namespace FFXIVClientStructs.FFXIV.Client.UI.Agent;
+
+[Agent(AgentId.GcArmyExpedition)]
+[StructLayout(LayoutKind.Explicit, Size = 0x48)]
+public unsafe struct AgentGcArmyExpedition
+{
+    [FieldOffset(0x00)] public AgentInterface AgentInterface;
+
+    [FieldOffset(0x28)] public GcArmyExpeditionData* ExpeditionData;
+    
+    [FieldOffset(0x40)] public int SelectedTab;
+    [FieldOffset(0x44)] public int SelectedRow;
+}
+
+[StructLayout(LayoutKind.Explicit, Size = 0x920)]
+public unsafe struct GcArmyExpeditionData
+{
+    [FieldOffset(0x10)] public int NumEntries;
+    [FieldOffset(0x18)] public MissionInfo* MissionInfoArray;
```

Comment:
i think this should be a FixedSizeArray instead of a pointer?

Reply:
I don't know what it's actual size is, I suspect it's variable, because it includes all the items in your armory aswell, but for me there wasn't dozens of blank spaces after the struct.

Reply:
It's strange at least in the memory it looks like it has pre-allocated several more entries.
Only the first 18 are used, so I'll mark it as a fixedsizearray(18)